build.sh

```diff
@@ -3,10 +3,15 @@ set -e
 
 # To build with tests, install cmocka-dev and define EXTRA_CMAKE_FLAGS as:
 # EXTRA_CMAKE_FLAGS=-DUNIT_TESTING=ON
+# Also make sure that gcc-8 is used when building the test suite.  If it isn't, add (e.g.) -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8
+# to EXTRA_CMAKE_FLAGS.  See example below:
+# EXTRA_CMAKE_FLAGS="-DUNIT_TESTING=ON -DCMAKE_VERBOSE_MAKEFILE=1 -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8"
```

Comment:
I only added VERBOSE to debug compile time issues with my setup.  I included it in my comment in case others were wondering how to print out compile and link time information from the make process.